src/components/navBar/navBar.module.scss

```diff
@@ -79,4 +78,104 @@ $theme-active: #87d870;
         color: $white;
         font-weight: 600;
     }
+
+    .hamburgerIcon {
+        display: none;
+    }
+    .mobile_menu > li {
+        list-style: none;
+    }
+    .navLinks .mobile_menu {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        gap: 1rem;
+    }
+    .active {
+        color: $theme-active;
+        text-decoration: underline;
+        text-decoration-thickness: $thickness;
+        text-underline-offset: $offset;
+    }
+}
+@media (max-width: 970px) {
+    .navBar {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        position: relative;
+        .navLinks {
+            a {
+                padding: 10px 40px;
+                text-align: left;
+                color: $theme-blue;
+                display: none;
+            }
+        }
+        .hamburgerIcon {
+            display: inline-block;
+            height: 73px;
+            width: 73px;
+            padding: 18px;
+            cursor: pointer;
+            font-size: 20px;
+            color: $white;
+        }
+    }
+    .signInLink {
+        position: absolute;
+        top: 10px;
+        right: 20px;
+    }
```

Comment:
this .signInLink class is for the sign-in with GitHub button when the user is not logged in and

Reply:
explain this further please.

Reply:
This is basically for sign in button and this is given position absolute because it was getting removed from its position when I was trying to fix the navbar so that's why I did this.